job/server/src/main/java/alluxio/worker/job/task/TaskExecutorManager.java

```diff
@@ -201,6 +203,7 @@ public synchronized void notifyTaskFailure(long jobId, long taskId, Throwable t)
     }
     finishTask(id);
     LOG.info("Task {} for job {} failed: {}", taskId, jobId, errorMessage);
+    SAMPLING_LOGGER.info(Throwables.getStackTraceAsString(t));
```

Comment:
I think this message should have taskId and jobId, so we know what the stack is for?

Something like:
```java
SAMPLING_LOGGER.info("Stack trace for taskId: {} jobId: {} : {}", taskId, jobId, Throwables.getStackTraceAsString(t));```

Reply:
Thanks. Done.